inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -809,6 +810,7 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
                 std::make_shared<Builder::NodeConverter<::ngraph::op::TopKIE>>(),
                 std::make_shared<Builder::NodeConverter<::ngraph::op::Unsqueeze>>(),
                 std::make_shared<Builder::NodeConverter<::ngraph::op::TensorIterator>>(),
+                std::make_shared<Builder::NodeConverter<::ngraph::opset5::Loop>>(),
```

Comment:
Why you add node converter? Could you use visitorAPI?

Reply:
For now, the old NodeConverter of TensorIterator is used for Loop op with small specific (NodeConverter<Loop> just sets the name of the Loop operation), but this code has not been fully tested for Loop yet due to lack of plugin implementation and unsupported Const-> Results subgraphs. 
I guess it's not a blocker for this PR. NodeConverter logic can be moved to the new SpecificCreator for both TI and Loop ops in the next PR.